lib/config/presets/internal/workarounds.ts

```diff
@@ -82,7 +82,7 @@ export const presets: Record<string, Preset> = {
         allowedVersions: '/^(?:8|11|17|21|25|29)(?:\\.|$)/',
         description:
           'Limit Java runtime versions to LTS releases. To receive all major releases add `workarounds:javaLTSVersions` to the `ignorePresets` array.',
-        matchDatasources: ['docker', 'adoptium-java'],
+        matchDatasources: ['adoptium-java'],
         matchPackageNames: [
```

Comment:
our datasource only supports `java`, `java-jre` and `java-jdk` as package names.